pkg/client/src/app/App.tsx

```diff
@@ -13,8 +13,13 @@ import "@redhat-cloud-services/frontend-components-notifications/index.css";
 
 import { ConfirmDialogContainer } from "./shared/containers/confirm-dialog-container";
 import { BulkCopyNotificationsContainer } from "./shared/containers/bulk-copy-notifications-container";
+import { useFetchCookie } from "./queries/cookies";
+import keycloak from "./keycloak";
 
 const App: React.FC = () => {
+  // Automatically updates cookie
+  const { refetch } = useFetchCookie(keycloak.token);
```

Comment:
Not sure why this not used ?

Reply:
Just updated to remove these unused exports.